frontend/testing/cypress/src/integration/studio/designer.js

```diff
@@ -43,24 +43,46 @@ context('Designer', () => {
 
     // Add new page and ensure updated data is loaded
     designer.getAddPageButton().click();
+
     cy.wait('@postLayoutSettings').its('response.statusCode').should('eq', 200);
     cy.wait('@getLayoutSettings').its('response.statusCode').should('eq', 200);
 
-    // Verify navigation button exists in form
-    designer.getDroppableList().findByRole('listitem', { name: texts['ux_editor.component_navigation_buttons'] }).should('be.visible');
-
     // Add an input component
     designer.getToolbarItemByText(texts['ux_editor.component_input']).trigger('dragstart');
     designer.getDroppableList().trigger('drop');
     cy.wait(500);
-    designer.getDroppableList()
+    designer
+      .getDroppableList()
       .findAllByRole('listitem')
-      .then(($elements) => expect($elements.length).eq(2));
+      .then(($elements) => expect($elements.length).eq(1));
 
     // Delete components on page
     cy.deleteComponents();
   });
 
+  it('should add navigation buttons when adding more than one page', () => {
+    cy.intercept('GET', '**/app-development/layout-settings?**').as('getLayoutSettings');
+    cy.intercept('POST', '**/app-development/layout-settings?**').as('postLayoutSettings');
+
+    // Navigate to designerApp
+    cy.visit('/editor/' + designerAppId);
+    header.getCreateLink().click();
+    cy.ensureCreatePageIsLoaded();
+
+    // Add new page and ensure updated data is loaded
+    designer.getAddPageButton().click();
+    designer.getAddPageButton().click();
+
+    cy.wait('@postLayoutSettings').its('response.statusCode').should('eq', 200);
+    cy.wait('@getLayoutSettings').its('response.statusCode').should('eq', 200);
+
+    // Add an input component
+    cy.wait(500);
+    designer
+      .getDroppableList()
```

Comment:
Tror vi kan fjerne denne også? 😄 
```suggestion
```

Reply:
Vi burde egentlig se litt mer på denne selectoren `.findByRole('listitem', { name: `${texts['ux_editor.component_navigation_buttons']}` });`. Fordi denne treffer vel egentlig både i left-menu, men også inn i selve siden? Her burde vi egentlig se om vi klarer å se forskjell på komponenter som er inne i siden vs utsiden (side-menyen).

Reply:
Fjerner jeg den, så får jeg denne feilen her. Så tror vi må ha den.
<img width="409" alt="image" src="https://github.com/Altinn/altinn-studio/assets/133344438/417b662d-9f6a-418b-bc0a-e20be2707bc8">

Reply:
ah, tenkte ikke på at den gikk mot designer objektet. Hva med `cy.findByRole(-------)`? 🤔

Reply:
Når vi bruker getDroppableList, utfører vi en findByRole-operasjon innenfor droppableList. Så langt jeg kan se, representerer dette alle komponenter i venstre menyen. Vårt mål er å forsikre oss om at vi spesifikt henter den som tilhører Side 2.

Et alternativ er å først be om komponentene på Side 2 og deretter utføre findByRole innenfor denne delen.

Reply:
Ja, godt poeng! 
Oppdaterte det nå 😎